how2meet/utils.py

```diff
@@ -17,7 +18,7 @@ def __init__(self, base_url: str = BASE_URL) -> None:
         self.base_url = base_url
 
     @classmethod
-    async def get_events(cls) -> list:
+    async def get_events(cls) -> Union[dict, List[dict]]:
```

Comment:
```suggestion
    async def get_events(cls) -> dict | list[dict]:
```
Newer python (I think 3.10?) lets you use more "standard" operators instead of having to import from `typing` -- doesn't matter much, I just think its prettier lol

Reply:
I also prefer the "or" syntax to the "Union" syntax for typing, it's more similar to Typescript.